### ISSUE TITLE
Importblock got error BSON uint64

### ISSUE THREAD

**Issue by dome** [gochain-io/explorer#328]
I found error in  grabber  when import some block 
```
> 9:40AM INF checkParentForBlock Redownloading the block because it's corrupted or missing:=364932                                   
> 9:40AM FTL importBlock error="BSON has no uint64 type, and value is too large to fit correctly in an int64"     
```

**Comment by jmank88** [comment]
This error was addressed in #326.
@dome What version of the code are you using? Can you try with the latest?

**Comment by dome** [comment]
i use v0.4.81

**Comment by jmank88** [comment]
@r-gochain Any idea what is going on here?

**Comment by r-gochain** [comment]
this error has been solved in the most recent version - https://github.com/gochain-io/explorer/pull/326/files#diff-804150087bd7a41c439588e4beb9fa19L15


Try :stable tag

**Comment by r-gochain** [comment]
*checked both testnet and mainnet and don't see this error in logs, maybe ci failed to build the image

**Comment by jmank88** [comment]
The fix should have been available since `0.4.80`, and both `0.4.80` and `0.4.81` appear to have built and pushed back to github and docker repos properly.

@dome Are you sure you are on the latest, `0.4.81`?

**Comment by dome** [comment]
So sorry. my docker-compose use wrong docker image
